app/bot/openai.go

```diff
@@ -18,38 +19,73 @@ type OpenAIClient interface {
 	CreateChatCompletion(context.Context, openai.ChatCompletionRequest) (openai.ChatCompletionResponse, error)
 }
 
+// OpenAIConfig contains parameters for OpenAI bot
+type OpenAIConfig struct {
```

Comment:
Config here won't be my preferred term as config is usually about .. configuration (files). Probably calling it OpenAIParams will be better